themes/vue/source/css/_sidebar.styl

```diff
@@ -74,7 +74,7 @@
     -webkit-transform: translate(-280px, 0)
     transform: translate(-280px, 0)
     .sidebar-inner
-      padding: 50px 10px 10px 20px
```

Comment:
Can we take this opportunity to increase the bottom padding of `.sidebar-inner` to something like 60px as well? The current value (10px) makes it almost possible to tap the last menu item on mobile devices.

![image](https://user-images.githubusercontent.com/8056274/64961741-6dc79080-d896-11e9-97c1-dfb45cd820cd.png)

Reply:
@phanan I've updated the bottom padding to 30px since the 60px was to account for the header and it would create too much gap.

![Screen Shot 2019-09-16 at 10 18 10 AM](https://user-images.githubusercontent.com/4836334/64965595-51fac500-d86b-11e9-96ae-d31bae71c814.png)

Let me know if you think we still need more space!